pkg/nsx/services/firewall.go

```diff
@@ -125,8 +142,24 @@ func (service *SecurityPolicyService) buildPolicyGroup(obj *v1alpha1.SecurityPol
 		return nil, "ANY", nil
 	}
 
+	targetGroupCount := 0
+	count := 0
+	var err error = nil
 	for i, target := range appliedTo {
-		service.updateTargetExpressions(obj, &target, &policyGroup, i)
+		count, err = service.updateTargetExpressions(obj, &target, &policyGroup, i)
+		if err == nil {
+			targetGroupCount += count
+		} else {
+			return nil, "", err
+		}
+	}
+	log.V(1).Info("Build policy target group criteria count:", "targetGroupCount", targetGroupCount)
```

Comment:
build -> build
And don't append `:` in the structure log

Reply:
remove all append : in the structure log.